contrib/fastfetch/template.py

```diff
@@ -0,0 +1,62 @@
+pkgname = "fastfetch"
+pkgver = "2.8.10"
+pkgrel = 0
+build_style = "cmake"
+configure_args = ["-DCMAKE_BUILD_TYPE=RelWithDebInfo", "-DBUILD_TESTS=ON"]
```

Comment:
```suggestion
configure_args = ["-DBUILD_TESTS=ON"]
```

this doesn't do anything

Reply:
(the -debug in the version is based on NDEBUG which you can put into cflags)